src/main/resources/messages_cn_BO.properties

```diff
@@ -0,0 +1,1366 @@
+metadata.trapped=བཟུང་བ།
+metadata.remove=གནས་ཚུལ་ཞིབ་ཕྲ་སུབ་པ།
+
+#optimize
+optimize.title=ལེགས་སྒྲིག
+optimize.header=ལེགས་སྒྲིག
+optimize.submit=ལེགས་སྒྲིག
+optimize.selectText=PDF ཡིག་ཆ་གདམ་ག
+optimize.imageQuality=པར་རིས་སྤུས་ཚད།
+optimize.colorImageQuality=ཚོན་ལྡན་པར་རིས་སྤུས་ཚད།
+optimize.grayImageQuality=སྐྱ་མདོག་པར་རིས་སྤུས་ཚད།
+optimize.bwImageQuality=དཀར་ནག་པར་རིས་སྤུས་ཚད།
+
+
+
+###########################
+#                         #
+#       WEB PAGES         #
+#                         #
+###########################
+#login
+login.title=ནང་འཛུལ།
+login.header=ནང་འཛུལ།
+login.signin=ནང་འཛུལ།
+login.rememberme=ང་རང་དྲན་པ།
+login.invalid=སྤྱོད་མཁན་མིང་ངམ་གསང་ཚིག་ནོར་འདུག
+login.locked=ཁྱེད་ཀྱི་ཐོ་མིང་བཀག་སྡོམ་བྱས་འདུག
+login.signinTitle=ནང་འཛུལ་གནང་རོགས།
+login.ssoSignIn=གཅིག་སྒོ་ནང་འཛུལ་བརྒྱུད་ནས་ནང་འཛུལ།
+login.oauth2AutoCreateDisabled=OAuth2 རང་འགུལ་སྤྱོད་མཁན་གསར་བཟོ་བཀག་སྡོམ་བྱས་འདུག
+login.oauth2AdminBlockedUser=ད་ལྟ་ཐོ་འགོད་མ་བྱས་པའི་སྤྱོད་མཁན་གྱི་ཐོ་འགོད་དང་ནང་འཛུལ་བཀག་སྡོམ་བྱས་འདུག དོ་དམ་པར་འབྲེལ་བ་གནང་རོགས།
+login.oauth2RequestNotFound=ར་སྤྲོད་རེ་ཞུ་རྙེད་མ་བྱུང་།
+login.oauth2InvalidUserInfoResponse=སྤྱོད་མཁན་ཆ་འཕྲིན་ལན་འདེབས་ནུས་མེད།
+login.oauth2invalidRequest=རེ་ཞུ་ནུས་མེད།
+login.oauth2AccessDenied=སྤྱོད་དབང་ཕྱིར་འཐེན།
+login.oauth2InvalidTokenResponse=Token ལན་འདེབས་ནུས་མེད།
+login.oauth2InvalidIdToken=Token ནུས་མེད།
+login.relyingPartyRegistrationNotFound=བརྟེན་ས་ཕྱོགས་ཀྱི་ཐོ་འགོད་རྙེད་མ་བྱུང་།
+login.userIsDisabled=སྤྱོད་མཁན་བཀག་སྡོམ་བྱས་འདུག ནང་འཛུལ་བཀག་སྡོམ་བྱས་འདུག དོ་དམ་པར་འབྲེལ་བ་གནང་རོགས།
+login.alreadyLoggedIn=ཁྱེད་རང་
+login.alreadyLoggedIn2=སྒྲིག་ཆས་ནང་ནང་འཛུལ་བྱས་ཟིན། སྒྲིག་ཆས་ནས་ཕྱིར་འབུད་བྱས་རྗེས་ཡང་བསྐྱར་ཚོད་ལྟ་བྱེད་རོགས།
+login.toManySessions=ཁྱེད་ལ་འབྲེལ་མཐུད་མང་དྲགས་འདུག སྒྲིག་ཆས་འགའ་ཤས་ནས་ཕྱིར་འབུད་བྱས་རྗེས་ཡང་བསྐྱར་ཚོད་ལྟ་བྱེད་རོགས།
+
+
+
+#addPageNumbers
+addPageNumbers.title=ཤོག་གྲངས་སྣོན་པ།
+addPageNumbers.header=ཤོག་གྲངས་སྣོན་པ།
+addPageNumbers.selectText.1=PDF ཡིག་ཆ་འདེམས་རོགས།：
+addPageNumbers.selectText.2=མཐའ་མཚམས་ཆེ་ཆུང་།
+addPageNumbers.selectText.3=གནས་ས།
+addPageNumbers.selectText.4=འགོ་འཛུགས་ཨང་གྲངས།
+addPageNumbers.selectText.5=ཤོག་གྲངས་སྣོན་དགོས་པའི་ཤོག་ངོས། སྔོན་སྒྲིག་ནི་"ཚང་མ་"ཡིན། 1-5 ཡང་ན་ 2,5,9 སོགས་ཀྱང་འཇུག་ཆོག
+addPageNumbers.selectText.6=རང་མོས་ཡི་གེ
+addPageNumbers.customTextDesc=རང་མོས་ཡི་གེ
+addPageNumbers.numberPagesDesc=ཤོག་གྲངས་སྣོན་དགོས་པའི་ཤོག་ངོས། སྔོན་སྒྲིག་ནི་"ཚང་མ་"ཡིན། 1-5 ཡང་ན་ 2,5,9 སོགས་ཀྱང་འཇུག་ཆོག
+addPageNumbers.customNumberDesc=སྔོན་སྒྲིག་ནི་ {n} ཡིན། "ཤོག་ངོས་ {n} / ཁྱོན་ {total} ཤོག"、"ཡི་གེ-{n}"、"{filename}-{n}" སོགས་ཀྱང་འཇུག་ཆོག
+addPageNumbers.submit=ཤོག་གྲངས་སྣོན་པ།
+addPageNumbers.fontSize=ཡིག་གཟུགས་ཆེ་ཆུང་།
+addPageNumbers.fontName=ཡིག་གཟུགས་མིང་།
+addPageNumbers.preview=སྔོན་ལྟ།
+addPageNumbers.settings=སྒྲིག་འགོད།
+addPageNumbers.startingNumber=འགོ་འཛུགས་ཨང་གྲངས།
+addPageNumbers.firstPage=ཤོག་ངོས་དང་པོ་ནས་འགོ་འཛུགས།
+addPageNumbers.lastPage=ཤོག་ངོས་མཐའ་མ་བར་དུ།
+addPageNumbers.position=གནས་ས།
+addPageNumbers.position.bottomCenter=འོག་དཀྱིལ།
```

Comment:
Lots of things here added (by ai?) randomly that DO NOT exist in the original file

The order of translations and files must also be maintained and they are not in the same positions as they are in 
https://github.com/Stirling-Tools/Stirling-PDF/blob/main/src/main/resources/messages_en_GB.properties